week-3/F-array-forEach/exercise.js

```diff
@@ -3,11 +3,31 @@
   - For multiples of 3 print “Fizz” instead of the number 
   - For the multiples of 5 print “Buzz”. 
   - For numbers which are multiples of both 3 and 5 print “FizzBuzz”
-
   An array with numbers 0-15 has been provided.
 */
 
 var arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+var print = arr
+  .map(function(num) {
+    if (num == 0) {
+      return num;
+    }
+    if (num % 3 == 0 && num % 5 == 0) {
+      return "FizzBuzz";
+    }
+    if (num % 3 == 0) {
+      return "Fizz";
+    }
+    if (num % 5 == 0) {
+      return "Buzz";
+    } else {
+      return num;
+    }
+  })
+  .forEach(function(name, index) {
+    console.log(name);
+  });
+console.log(print);
 
```

Comment:
Have another look at the expected output vs what your code produces - does the expected answer include 0?

Reply:
var print = arr.map(function(num) {
  if (num == 0) {
    return num;
  }
  if (num % 3 == 0 && num % 5 == 0) {
    return "FizzBuzz";
  }
  if (num % 3 == 0) {
    return "Fizz";
  }
  if (num % 5 == 0) {
    return "Buzz";
  } else {
    return num;
  }
});
print.forEach(function(name) {
  console.log(name);
});

Reply:
yes i noticed that and i fixed it but i didn't commit it yet but this is how  i did it